src/components/FactionButton.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
 import {TouchableOpacity, Image, View, Text, StyleSheet} from 'react-native';
 
-export const FactionButton = ({image, label, borderColor}) => {
+export const FactionButton = ({image, label, color, isSelected, ...props}) => {
```

Comment:
suggestion: here you can replace all this object with a single one (props), and when you call something that comes from properties you can do something like {props.label} (in case of text)